repository/.project

```diff
@@ -0,0 +1,27 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Do you git-ignore this one as well?

Reply:
.project files are needed for eclipse-based IDEs, and it is good to include them in the repo so that it takes much less time to setup the work environment with .projects files ready. Otherwise, the developer/tester will have to establish the work environment with lots of .project file manually generated again and again.